proto/interchain_security/ccv/v1/shared_consumer.proto

```diff
@@ -0,0 +1,146 @@
+syntax = "proto3";
+
+package interchain_security.ccv.v1;
+
+option go_package = "github.com/cosmos/interchain-security/v3/x/ccv/types";
+
+import "tendermint/abci/types.proto";
+import "ibc/lightclients/tendermint/v1/tendermint.proto";
+import "ibc/core/channel/v1/channel.proto";
+import "google/protobuf/duration.proto";
+import "gogoproto/gogo.proto";
+import "interchain_security/ccv/v1/wire.proto";
+import "google/protobuf/timestamp.proto";
+
+//
+// Note any type defined in this file is referenced/persisted in both the consumer and provider CCV modules,
```

Comment:
TLDR most consumer protos are persisted on provider, so most ended up in this file

Reply:
I see where you're coming from, but from a pure dependency perspective, the consumer's genesis state struct is referenced by both the consumer and provider modules. Any type that's a field in that struct must also be referenced by both modules, even if nil. 

There's some ways we could untangle these cross dependancies, like creating a new struct that'd be spit out of `MakeConsumerGenesis` for the provider, and making that a field in the consumer's true GenesisState struct. But there's complexities in how the consumer genesis JSON would be constructed. 

This PR is mostly intended to illuminate dep issues rather than fix them all, and it should stay as a true refactor